phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/standard/main.less

```diff
@@ -248,20 +248,25 @@
 /*******************************************************/
 /*********************** library ***********************/
 /*******************************************************/
-.shortcutButton {
-  -fx-background-color: transparent;
-  -fx-label-padding: 2.5em 0 0 0;
-  -fx-background-size: 3em;
-  -fx-background-repeat: no-repeat;
-  -fx-background-position: top;
-  -fx-cursor: hand;
+.shortcut-control-button-group {
```

Comment:
Is main.less the only style file which uses this?

Reply:
What do you mean by uses?
The other styles don't override its properties, therefore I didn't add it to the other styles.

Reply:
Yes, that was my question.

Reply:
Should I add it also to the other styles (as an empty block statement) or is it ok as is?